### DISCUSSION TITLE
2023년 03월 23일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#18] [General]
# 2023년 03월 23일

⚖️ 공부한 것, 일기를 적는 개인 위키입니다 인덱스 페이지: [ wiki-index ] 이번 달의 TIL: [ 2023-03|2023년 3월의 TIL✨ ] 현재 자주 쓰는 글들: [ wooteco|우아한테크코스 ]

https://cyma-s.github.io/2023-03-23/

**Comment by ezzanzzan** [comment]
수고했어 로베야

**Comment by Choi-JJunho** [comment]
미움받을 용기를 장전하면 눈치볼 필요가 없죠~
베로 화이팅!

**Reply by Cyma-s** [reply]
`'일발 장전'`

**Comment by greeng00se** [comment]
오늘도 화이팅~

**Reply by Cyma-s** [reply]
선플 감사합니다~?

**Comment by be-student** [comment]
세줄요약 해주세요~

**Reply by Cyma-s** [reply]
알아서 해주세요~
